crates/kafka-source/src/source.rs

```diff
@@ -3,14 +3,15 @@ use async_std::channel::{self, Sender};
 use async_std::task::spawn;
 use async_trait::async_trait;
 use fluvio::{dataplane::record::RecordData, Offset, RecordKey};
-use fluvio_connector_common::tracing::{info, trace};
+use fluvio_connector_common::tracing::{error, info, trace};
 use fluvio_connector_common::Source;
 use futures::{stream::LocalBoxStream, StreamExt};
 use kafka::consumer::{Consumer, FetchOffset, GroupOffsetStorage};
 
 use crate::config::KafkaConfig;
 
 const CHANNEL_BUFFER_SIZE: usize = 10000;
+const DEFAULT_FETCH_MAX_BYTES_PER_PARTITION: i32 = 1024 * 1024;
```

Comment:
Is this good for a default? The Kafka library uses 32 × 1024.

Reply:
should be good